keel-sql/src/main/resources/db/changelog/20201016-create-lifecycle-events.yml

```diff
@@ -0,0 +1,151 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-lifecycle-events
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: lifecycle
```

Comment:
I'd suggest being more explicit with the table name and calling it `lifecycle_event`